### PR TITLE
fix: remove pre-commit hooks from checklist

### PR DESCRIPTION
**Pull request by signekb**
we will not implement pre-commit hooks after all

### CODE REVIEW

**Review by K-Beicher — DISMISSED** [review]
Should the 'Build' par of the code section stay, or is that linked with the hooks?

**Comment by signekb** [comment]
Not entirely sure what is meant by *build* here, actually? :)

File: .github/PULL_REQUEST_TEMPLATE/pull-request-code.md [hunk 1]
```diff
@@ -54,5 +54,4 @@ Closes Issue ...
 - [ ] Tests accompany or reflect changes to the code
 - [ ] Tests ran and passed locally
 - [ ] Ran the linter and formatter
-- [ ] Build and pre-commit hooks have passed locally
```
Comment (lwjohnst86) [inline .github/PULL_REQUEST_TEMPLATE/pull-request-code.md:57]: I think @K-Beicher meant this build here. I think we could keep this without the precommit text

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]

**Comment by K-Beicher** [comment]
@lwjohnst86 Talking about this, should I look at making one PR document?  In which case we won't need to remove them here...

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**